.github/workflows/test-with-edm.yml

```diff
@@ -15,9 +15,14 @@ jobs:
   test-with-edm:
     strategy:
       matrix:
-        os: [macos-latest]
+        os: [macos-latest, windows-latest]
         toolkit: ['pyqt', 'pyqt5', 'pyside2', 'wx']
     runs-on: ${{ matrix.os }}
+    env:
+      # Set root directory, mainly for Windows, otherwise 'pip install'
+      # complains because the Python environment is in C: drive whereas
+      # the source is checked out in D: drive.
+      EDM_ROOT_DIRECTORY: ${{ github.workspace }}/.edm
```

Comment:
i'm still not entirely sure what this comment is trying to communicate - i don't understand how setting the `EDM_ROOT_DIRECTORY` prevents `pip` from complaining. I looked at the output of a few jobs to see if i can find the motivating issue but i can't find it right now.

Reply:
I will fix the comment.
For reference, this is one of the logs that shows the error: 
[log.txt](https://github.com/enthought/enable/files/5803611/log.txt)

The error looks like this:
```
2021-01-11T10:33:14.8632545Z ERROR: Exception:
2021-01-11T10:33:14.8633351Z Traceback (most recent call last):
2021-01-11T10:33:14.8634326Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\site-packages\pip\_internal\cli\base_command.py", line 186, in _main
2021-01-11T10:33:14.8635304Z     status = self.run(options, args)
2021-01-11T10:33:14.8636210Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\site-packages\pip\_internal\commands\install.py", line 404, in run
2021-01-11T10:33:14.8637152Z     use_user_site=options.use_user_site,
2021-01-11T10:33:14.8638083Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\site-packages\pip\_internal\req\__init__.py", line 71, in install_given_reqs
2021-01-11T10:33:14.8638937Z     **kwargs
2021-01-11T10:33:14.8639798Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\site-packages\pip\_internal\req\req_install.py", line 829, in install
2021-01-11T10:33:14.8640626Z     scheme=scheme,
2021-01-11T10:33:14.8641555Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\site-packages\pip\_internal\operations\install\legacy.py", line 123, in install
2021-01-11T10:33:14.8642829Z     os.path.relpath(prepend_root(filename), egg_info_dir)
2021-01-11T10:33:14.8643604Z   File "C:\Users\runneradmin\.edm\envs\enable-test-3.6-pyqt\lib\ntpath.py", line 584, in relpath
2021-01-11T10:33:14.8644269Z     path_drive, start_drive))
2021-01-11T10:33:14.8644834Z ValueError: path is on mount 'D:', start on mount 'C:'
```

Reply:
Hope the comment is clearer now, please take a look.

Reply:
yep. much better. thanks!

Reply:
Thank you!  Comments on obscure comments are very much appreciated - because if they are not clear, it is worse than not having any.